Dockerfile.template.erb

```diff
@@ -77,8 +77,15 @@ ENV TINI_VERSION=0.18.0
 RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 
 # Fluentd depends on cool.io whose fat gem is only available for Ruby < 2.5, so need to specify --platform ruby when install Ruby > 2.5 and install msys2 to get dev tools
-RUN choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
-&& choco install -y msys2 --version 20210604.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby26\msys64'"
+#RUN choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
+#&& choco install -y msys2 --version 20210604.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby26\msys64'"
+
+# For avoiding stalling with docker build, we use snapshot version of msys2 until newer msys2 will be shipped.
+# NOTE: msys2 must be extracted to c:\ruby26\msys64. It should be done with -oC:\ruby26 option.
+RUN powershell -Command "Invoke-WebRequest 'https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-20210628.sfx.exe' -OutFile 'msys2-base-x86_64-20210628.sfx.exe'" \
```

Comment:
I'm not against to use nightly build of msys2 installer but this nightly installer URL can be used permanently?
Dockerfile should be reproducible build.

Reply:
> I'm not against to use nightly build of msys2 installer but this nightly installer URL can be used permanently?

Ah, it should use `-latest` in this case.

Reply:
> > I'm not against to use nightly build of msys2 installer but this nightly installer URL can be used permanently?
> 
> Ah, it should use `-latest` in this case.

Yep, that's right. But ensuring reproducible docker build is thrown away when using `-latest` suffixed nightly installer.
This unresolved question still exists.

Reply:
But it's okay for me to use nightly `-latest` suffixed MSYS2 installer.
When we'll push built images on DockerHub, users can always derive their images from our pushed nightly used images.

Reply:
![image](https://user-images.githubusercontent.com/225841/124054701-7c783b00-da5d-11eb-8c0d-eb993ed42cff.png)

At least, the nightly tag is kept since last year, it doubts whether  -latest installer will be thrown away :thinking: 
(Instead, I agree that it may be possible that the nightly installer will not be updated for a while)